std/node/process_test.ts

```diff
@@ -120,3 +123,33 @@ Deno.test({
     assertEquals(typeof env.PATH, "string");
   },
 });
+
+Deno.test({
+  name: "process.stdin",
+  fn() {
+    assertEquals(typeof process.stdin.rid, "number");
+    assertEquals(process.stdin.rid, Deno.stdin.rid);
```

Comment:
In Node there isn't a process.stdin.rid

Reply:
Okay! I’ll get peep everything on node.process and replicate it more explicitly instead of spreading denos stdio.